package.json

```diff
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/hicommonwealth/chain-events#readme",
   "dependencies": {
+    "@commonwealth/chain-events": "file:.yalc/@commonwealth/chain-events",
```

Comment:
We should remove this / figure out why this exists.

Reply:
Removed!